app/components/sections/blog-section.tsx

```diff
@@ -0,0 +1,39 @@
+import clsx from 'clsx'
+import * as React from 'react'
+import {H2} from '../title'
+import {ArrowButton} from '../arrow-button'
+import {Grid} from '../grid'
+import {ArticleCard, ArticleCardProps} from '../article-card'
+
+interface BlogSectionProps {
+  articles: ArticleCardProps[]
+}
+
+function BlogSection({articles}: BlogSectionProps) {
+  return (
+    <Grid>
+      <div className="flex flex-col col-span-full mb-10 space-y-10 lg:flex-row lg:items-center lg:justify-between lg:space-y-0">
+        <div className="space-y-2 lg:space-y-0">
+          <H2>Most popular from the blog.</H2>
+          <H2 variant="secondary">Probably the most helpful aswell.</H2>
+        </div>
+
+        <ArrowButton direction="right">See the full blog</ArrowButton>
+      </div>
+
+      {/*This should be a horizontal scroller */}
```

Comment:
note to self, no it shouldn't. I've talked to the designer about it, and the code has already been adjusted. Forgot to remove the comment :grimacing:. On desktop, it shows 3 columns. On tablet 2, and on mobile 2 rows.